models/list/list.go

```diff
@@ -74,7 +76,7 @@ func (u *ListStruct) Initialize() {
 	u.database = interfaceDB
 }
 
-func (u *ListStruct) Save(repo *repository.RepositoryStruct, OauthToken *sql.NullString) bool {
+func (u *ListStruct) Save(repo *repository.RepositoryStruct, OauthToken *sql.NullString) (e error) {
```

Comment:
__[golint]__
* [golint] exported method ListStruct.Save should have comment or be unexported